server/tnt_workdir/init.lua

```diff
@@ -145,5 +145,26 @@ else
      print('space already exists: clusters_cache')
 end
 
+if not box.space.metro then
+    local metro = box.schema.space.create('metro')
+    metro:create_index('primary', {
+                        type = 'TREE',
+                        parts = {1, 'NUM'},
+    })
+    metro:create_index('spatial', {
+        type = 'RTREE',
+        parts = { 2, 'ARRAY' },
+        unique = false,
+    })
+    metro:create_index('secondary', {
```

Comment:
Will be great to have some sort of comment about this index (Looks like town_id ?)